ldapauthenticator/ldapauthenticator.py

```diff
@@ -663,11 +667,7 @@ async def authenticate(self, handler, data):
                 )
                 if found:
                     ldap_groups.append(group)
-                    # we currently only use this in check_allowed,
-                    # so we stop here, as only one match is relevant
-                    # if all groups are needed (e.g. for manage_groups)
-                    # we should keep fetching membership
-                    break
+                    # Returned in auth_state, so fetch the full list
```

Comment:
An alternative, if we don't want to iterate through all groups, is to change the property to `is_in_allowed_groups: true|false`